src/main/scala/com/dwolla/lambda/cloudflare/record/CloudFlareDnsRecordHandler.scala

```diff
@@ -52,15 +52,14 @@ class CloudflareDnsRecordHandler(implicit ec: ExecutionContext) extends ParsedCl
       .map(cloudFlareApiClient)
   }
 
-  private def parseDtoFrom(id: Option[String], resourceProperties: Map[String, JValue]) = {
+  private def parseRecordFrom(resourceProperties: Map[String, JValue]) = {
     val ttl: Option[String] = resourceProperties.get("TTL")
     val proxied: Option[String] = resourceProperties.get("Proxied")
 
-    DnsRecordDTO(
```

Comment:
Moving away from `DnsRecordDTO` here makes me a little nervous because I originally had some issues with correctly parsing the JSON input into the Lambda and from CloudFlare, but I think this looks ok—it looks like every use was being implicitly converted to `UnidentifiedDnsRecord` anyway. Hopefully we're not missing anything. 🙂